### ISSUE TITLE
Add term delayed distal femoral epiphyseal ossification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10524]
**Preferred term label:**
Delayed distal femoral epiphyseal ossification

**Synonyms**
Delayed ossification of the distal femoral epiphysis - Delayed ossification distal femoral epiphyses

**Definition (free text, please give PubMed ID)**
Developmental delay of ossification of the distal epiphysis of the femur. https://doi.org/10.1002/oa.3240

**Parent term (use hpo.jax.org/app)**
Delayed epiphyseal ossification HP:0002663

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achondrogenesis type 2 (omim200610), for example PMID24144387

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000866
- label: Delayed distal femoral epiphyseal ossification
- parents: [Abnormality of the distal femoral epiphysis (HP:0010590)](https://hpo.jax.org/app/browse/term/HP:0010590)
- pmid: 
- def: Developmental delay of ossification of the distal epiphysis of the femur.
- comment: The distal femoral epiphysis (DFE), is the first of the long bone epiphyses to ossify and appears before the proximal tibial epiphysis (PTE). Around birth, the DFE consists of a rounded or oval nucleus with pitting and osseous circumvolutions. These epiphyses are maturation indicators marking the different stages of progression of bone maturation. They appear in a constant and defined order, which makes the infracranial sequence of maturation useful for age estimation of immature individuals.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10524
- synonyms:
